src/style.css

```diff
@@ -1 +1,46 @@
 /* add styles here */
+html,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 :first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 1),
+.stars--5 :nth-child(-n + 2),
+.stars--5 :nth-child(-n + 3),
+.stars--5 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
```

Comment:
There seems to be a mistake in your CSS selectors. The `:nth-child(-n + 1)` selector in the `.stars--5` rule will only select the first child, but you probably intended to select all children. To select all children, you can simply use the `:nth-child(n)` selector. Also, there's no need to repeat the same selector for each child. You can combine them into a single rule: `.stars--5 :nth-child(n)`.